estate/models/estate_property.py

```diff
@@ -0,0 +1,41 @@
+from odoo import models, fields
+
+class estate_property(models.Model):
+    _name = "estate.property"
+    _description = "estate property"
+
+    name= fields.Char("Plan Name", required = True ) # required make property not nullable 
+    description =  fields.Text("Plan desc")
+    postcode = fields.Char('postcode')
+    date_availability = fields.Date(copy=False,default=fields.Date.add(fields.Date.today() , months=3))  
```

Comment:
There must be a single space after every comma `,` .
```suggestion
    date_availability = fields.Date(copy=False, default=fields.Date.add(fields.Date.today(), months=3))  
```